const service = require("./service")

async function main() {
    try {

    } catch (error) {
        console.error(`error interno`.error)
    }
}